### PR TITLE
Adjust Colours.

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

- Change the lightness of the component colours to match Figma.
- Remove the background image from tailwind.css.
- Alter the code editor background to be white.
- Make the selection colour much brighter in the dashboard.

image

image

image

### Important Notes

Removed the `is-hidden-file` package from the project manager shim. It seems to be unreliable in Win11 and as just for local testing made just do dot file logic.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

File: distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso [hunk 1]
```diff
@@ -863,3 +866,13 @@ fits_in_range start end date =
...
     if that.get "type" == "Date" && ["year", "month", "day"].all that.contains_key then Date.new (that.get "year") (that.get "month") (that.get "day") else
         Error.throw (Illegal_Argument.Error "Invalid JS_Object for Date.from.")
+
+## PRIVATE
+private make_day_picker cache=Nothing =
+    year = cache.if_not_nothing <| cache "year"
+    month = cache.if_not_nothing <| cache "month"
+    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+    max = if month.is_nothing then 31 else
+        if month != 2 || year.is_nothing then days[month - 1] else
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso]: `days[month - 1]` isn't valid Enso code, is it?

I mean, I guess it is a function call `days x` where `x` is a Vector: `x = [month - 1]`. But I don't think that's what you meant.

Comment (radeusgd) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso]:
```suggestion
        if month != 2 || year.is_nothing then days.at (month - 1) else
```

Comment (radeusgd) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso]: Very cool picker in general.

But the logic is non-trivial and we can see above there was an error in the code. Both of these indicate that we should have some tests for this code. Let's test that the leap year code paths all work correctly.

**Review by radeusgd — APPROVED** [review]
Changes look good, but the error in the widget suggests we should have an unit test for it. Maybe we can do something like `*_Widgets_Spec.enso`?

**Review by kazcw — APPROVED** [review]

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**